src/week5/ReverseCharacters.java

```diff
@@ -0,0 +1,25 @@
+package week5;
+
+public class ReverseCharacters {
+    public static void main(String[] args) {
+
+        reverseChar("Sofia");
```

Comment:
why reverse char

Reply:
name of the method to reverse a character